src/index.css

```diff
@@ -42,7 +42,6 @@ button {
   font-family: inherit;
   border: none;
   border-radius: var(--borderRadius);
-  background-color: var(--text);
   color: #ffffff;
```

Comment:
Text color should be moved to the individual buttons

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix